### ISSUE TITLE
Insert Images; Path Broken

### ISSUE THREAD

**Issue by Ph4te** [turnermm/ckgedit#261]
Hi,
I'm using ckgedit  2017-02-18 on Debian Jessie and it shows the wrong media path when I try to insert an image to the wiki. To try this setting, I checked the winstyle-option and since then, Links in the media manager appear like `/wiki/data/media/picture.png`. (It looks like [this](https://github.com/turnermm/ckgedit/issues/228) one. ) Before I checked this option, all links would appear like `/var/www/dokuwiki/lib/plugins/ckgedit/fckeditor/userfiles/picture.png`.
Unchecking the winstyle-option doesn't restore the old behaviour.

I can upload new Images to the server and I can delete them, I just cannot insert them to any Wikipages.
There is a copy of the .htaccess.security file in both /var/www/dokuwiki/lib/plugins/ckgedit/fckeditor/userfiles and data/media. Of course i did rename them to .htaccess.

Do you have any idea how I could fix this?

**Comment by turnermm** [comment]
I'm afraid your desrciption of the problem is confusing to me.  This is correcto for the  `winstyle` setting:
    /wiki/data/media/picture.png. 
This is incorrect for any setting:
  /var/www/dokuwiki/lib/plugins/ckgedit/fckeditor/userfiles/picture.png.

Ths problem: "Unchecking the winstyle-option doesn't restore the old behaviour" is most likely a caching problem

**Comment by Ph4te** [comment]
> This problem: "Unchecking the winstyle-option doesn't restore the old behaviour" is most likely a caching problem

You're right. Even after emptying the cache in firefox, it didn't change the style of the links. Chrome displays them correctly.

My problem now basically is, that all links look like this and I cannot insert them:

![dokuwiki](https://cloud.githubusercontent.com/assets/15713814/23270954/f4fe618c-f9f5-11e6-84f4-cdbaeae05654.png)

**Comment by turnermm** [comment]
It's just that Chrome is starting with a fresh cache.  To refresh to cache in firfox hit CTRL-SHIFT-DELETE.  

I'm not sure where the misconfiguration is.  But go back to winstyle and try using the .htaccess.open for  your .htaccess.  Or substitue the .htaccess.security for the .htaccess file in the data directory, again renaming it .htaccess.

**Comment by Ph4te** [comment]
Hi, I tried that, but unfortunately it doesn't work. 
I activated winstyle and copied .htaccess.security from `/var/www/dokuwiki/lib/plugins/ckgedit/fckeditor/userfiles` to `/var/www/dokuwiki/data/media/` and renamed it to .htaccess. 
The links in the uploadmanager now look like `/wiki/data/media/picture.png`

The content of the .htaccess-File is

> #unix  -- do not change or move
> <IfModule mod_authz_host>
>     Require all denied
> </IfModule>
> <IfModule !mod_authz_host>
>     Order allow,deny
>     Deny from all
> </IfModule>
> 
> <FilesMatch "\.(gif|jpe?g|png)$"> 
> <IfModule mod_authz_host>
>     Require all granted
> </IfModule>
> <IfModule !mod_authz_host>
>     Order allow,deny
>     Allow from all
> </IfModule>
> </FilesMatch>
> 
> Options -Indexes

Do you have any other ideas?

**Comment by turnermm** [comment]
It may be a file permission issue,  but what I suggest is that you replace ckgedit with ckgdoku:
https://github.com/turnermm/ckgdoku/archive/master.zip
In your configuration options set to 'dokuwiki the settings for both the default media browser and which media browser can be used.  When you load an image into the image dialog preview, you shoudl then click on the image resize button ( the curved arrow next to the heioght/width boxes.

**Comment by turnermm** [comment]
ckgdoku is a version of ckgedit that uses Dokuwiki's file and media browsers, and so if they work for you thwne you switch to the native dokuwiki editor, then they should work here.

**Comment by Ph4te** [comment]
Ok, CKGDoku works as intended, thank you! 

There is just one problem. I install ckgdoku manually and it replaces ckgedit. But I can't update ckgdoku because it says the Download-URL was changed since the last download. It thinks the ckgdoku-URL is old and the ckgedit-URL is new and if I update the plugin, it replaces ckgdoku with ckgedit.
Do I have to manually change files on the server to permanently change this plugin? I tried uninstalling ckgedit before I did install ckgdoku, but it didn't change.
